cmd/collector/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
+EXPOSE 14267
+
+COPY collector-linux /go/bin/
+
+CMD ["/go/bin/collector-linux"]
```

Comment:
Not necessarily: I'm not quite sure how it would work for Docker Compose, but for Docker at the command line, Kubernetes and OpenShift, you can override the command line and set extra env vars.

So, the best approach is to specify only what's really required to get the binary running, so that you don't need to maintain two places with default values.

```bash
$ docker run -e MY_ENV_VAR=test -it jpkroehling/jaeger-agent bash
[root@86fc09d20bfc /]# echo $MY_ENV_VAR
test
```